### PR TITLE
PWN-5621 - Username in details

### PR DESCRIPTION
**Pull request by gslevinkov**
fix version: 2.0.1
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-5621

## Description of Work

- fix design on bottom sheet
- add username details for TRANSFER transaction

## Screenshots (Optional)

Screenshot #1 | Screenshot #2
:------------:|:-------------:
Screen Shot 2022-10-30 at 18 09 24| Screen Shot 2022-10-30 at 18 27 30 |

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/HistoryTransactionDetailsBottomSheetFragment.kt [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+import org.p2p.wallet.utils.setStatus
+import org.p2p.wallet.utils.showInfoDialog
+import org.p2p.wallet.utils.showUrlInCustomTabs
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+import org.p2p.wallet.utils.withTextOrGone
+
+private const val EXTRA_STATE = "EXTRA_STATE"
+
+class HistoryTransactionDetailsBottomSheetFragment :
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/HistoryTransactionDetailsBottomSheetFragment.kt:32]: renamed and added method to show username
sorry for the wrong diff

Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/HistoryTransactionDetailsBottomSheetFragment.kt:32]: i would even consider this fragment as a tech-debt and migrate it to recycler_view

**Review by 1sakov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetPresenter.kt [hunk 2]
```diff
@@ -1,104 +0,0 @@
...
-
-            val usdTotal = transaction.getFormattedAmount()
-            val total = transaction.getFormattedTotal()
-            showAmount(total, usdTotal)
-            showFee()
-            showBlockNumber(transaction.getBlockNumber())
-            showTransferView(transaction.getIcon())
-        }
-    }
-}
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetPresenter.kt:104]: Did you just rename this class and the git history broke here?

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetPresenter.kt:104]: yep, just renaming
you can see it in the commit history

**Review by 1sakov — APPROVED** [review]

**Pull request merged**